### ISSUE TITLE
Installation question

### ISSUE THREAD

**Issue by singhigh** [migda/timer24.net#2]
when i use cmd, it shows: PHP Parse error:  syntax error, unexpected 'class' (T_CLASS), expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in /home/wwwroot/domain/artisan on line 31

**Comment by migda** [comment]
Check your PHP version. Laravel 5.3 required PHP >= 5.6.4.

**Comment by singhigh** [comment]
PHP Fatal error:  require(): Failed opening required '/home/wwwroot/domain/bootstrap/../vendor/autoload.php' (include_path='.:/usr/local/php/lib/php') in /home/wwwroot/domain/bootstrap/autoload.php on line 17

did you lost vendor?

**Comment by migda** [comment]
Yes. Sorry about that. I added it to .gitignore. I am going to upload it soon.

**Comment by singhigh** [comment]
Thank you! and I am also making chinese language translation (https://github.com/singhigh/timer24.net/tree/master/resources/lang/zh-cn) but i have some puzzle and i have to install it to understand something, i love this code and i have some productive use in my job.

**Comment by migda** [comment]
Nice! I uploaded /vendor. Try it now.

**Comment by singhigh** [comment]
Hello, another error:
[ErrorException]                                                         
  Undefined property: Illuminate\Database\Schema\Blueprint::$smallInteger

**Comment by singhigh** [comment]
I am really sorry for my stupid:

RuntimeException in Encrypter.php line 43:
The only supported ciphers are AES-128-CBC and AES-256-CBC with the correct key lengths.
in Encrypter.php line 43
at Encrypter->__construct('', 'AES-256-CBC') in EncryptionServiceProvider.php line 27
at EncryptionServiceProvider->Illuminate\Encryption\{closure}(object(Application), array()) in Container.php line 746
at Container->build(object(Closure), array()) in Container.php line 644
at Container->make('encrypter', array()) in Application.php line 709
at Application->make('Illuminate\Contracts\Encryption\Encrypter') in Container.php line 864
at Container->resolveClass(object(ReflectionParameter)) in Container.php line 819
at Container->getDependencies(array(object(ReflectionParameter)), array()) in Container.php line 790
at Container->build('App\Http\Middleware\EncryptCookies', array()) in Container.php line 644
at Container->make('App\Http\Middleware\EncryptCookies', array()) in Application.php line 709
at Application->make('App\Http\Middleware\EncryptCookies') in Pipeline.php line 127
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 104
at Pipeline->then(object(Closure)) in Router.php line 644
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 618
at Router->dispatchToRoute(object(Request)) in Router.php line 596
at Router->dispatch(object(Request)) in Kernel.php line 267
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 104
at Pipeline->then(object(Closure)) in Kernel.php line 149
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 116
at Kernel->handle(object(Request)) in index.php line 54

**Comment by migda** [comment]
> [ErrorException] Undefined property: Illuminate\Database\Schema\Blueprint::$smallInteger

I forgot to name new column in `events`  table (`timezone`) in migration file. I fixed this.

**Comment by singhigh** [comment]
any modification on nginx configuration file?

like add:

location / {  
    try_files $uri $uri/ /index.php$is_args$args;
}

**Comment by migda** [comment]
I run it on Apache. I didn't try to run on nginx.

In readme file I forgot to mention about copy .env.example file, modify in it db connection and then generate key by php artisan:

cp .env.example .env
php artisan key:generate

**Comment by singhigh** [comment]
I did copy .eny and googled to generated the key, thank you! It will be much better if I can customized site name!

**Comment by migda** [comment]
Thank you for your suggestion. Maybe I should add new option in administration panel where admin could change some settings like site name.

For now: You can change website name in config/app.php on line 14 and in views files (where I didn't use Config::get('app.name') ;-/)

**Comment by singhigh** [comment]
Modified! And I will try to modify a chinese version, thank you! have a nice day

**Comment by migda** [comment]
You're welcome :). Have a nice day!
